starlette_jsonapi/resource.py

```diff
@@ -130,17 +134,24 @@ async def validate_body(self, partial=None) -> dict:
             raise JSONAPIException(status_code=400, errors=errors.get('errors'))
         return body
 
-    async def serialize(self, data: Any, many=False, *args, **kwargs) -> dict:
+    async def serialize(self, data: Any, many=False, paginate=False, *args, **kwargs) -> dict:
```

Comment:
We should have some way of controlling the pagination parameters here. What do you think of a special argument like `pagination_kwargs` (`paginator_kwargs`) which we can pass to `paginate_request` ?